### PR TITLE
Stablepools

### PR DESCRIPTION
**Pull request by kolebjak**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,6 +29,7 @@
     "@galacticcouncil/bridge": "0.0.7",
     "@galacticcouncil/math-lbp": "^0.1.3",
     "@galacticcouncil/math-liquidity-mining": "^0.1.3",
+    "@galacticcouncil/math-stableswap": "../HydraDX-wasm/packages/stableswap/",
```
Comment (vkulinich-cl) [inline package.json:32]: There is a math-stableswap released pkg. Can you use it instead?

Comment (kolebjak) [reply package.json:32]: Waiting for new version to be released. Latest is `0.1.4` which doesn't include changes I need.

File: src/components/AssetSelect/AssetSelectButton.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import { Icon } from "../Icon/Icon"
+import { Text } from "../Typography/Text/Text"
+import { theme } from "../../theme"
```
Comment (vkulinich-cl) [inline src/components/AssetSelect/AssetSelectButton.tsx:3]: use absolute path

File: src/components/AssetSelect/AssetSelectButton.tsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+import { getAssetName } from "../AssetIcon/AssetIcon"
+import { SSelectAssetButton } from "./AssetSelect.styled"
+import { ReactComponent as ChevronDown } from "assets/icons/ChevronDown.svg"
+import { u32 } from "@polkadot/types-codec"
+import { useAsset } from "../../api/asset"
```
Comment (vkulinich-cl) [inline src/components/AssetSelect/AssetSelectButton.tsx:8]: absolute path

File: src/components/Modal/Modal.styled.ts [hunk 4]
```diff
@@ -92,36 +111,9 @@ export const SContent = styled.div<{
   z-index: ${theme.zIndices.modal};
 
   display: flex;
+  flex-flow: column;
   overflow: hidden;
 
-  background: ${theme.colors.darkBlue700};
```
Comment (vkulinich-cl) [inline src/components/Modal/Modal.styled.ts:97]: what's a reason of removing the gradient border?

File: src/sections/pools/stablepool/StablePool.tsx [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+  reserves,
+}: Props) => {
+  const [isExpanded, setIsExpanded] = useState(false)
+  const isDesktop = useMedia(theme.viewport.gte.sm)
+
+  const { account } = useAccountStore()
+  const position = useTokenBalance(id.toString(), account?.address)
+
+  const amount = position?.data?.freeBalance ?? BN_0
+  const hasPosition = amount?.isGreaterThan(BN_0)
```
Comment (vkulinich-cl) [inline src/sections/pools/stablepool/StablePool.tsx:40]: amount won't be an undefined

File: src/sections/pools/stablepool/details/PoolDetails.tsx [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+          <div sx={{ flex: "column", gap: 5 }}>
+            <MultipleIcons
+              icons={assets.map((asset) => ({
+                icon: getAssetLogo(asset.symbol),
+              }))}
+            />
+            <div sx={{ flex: "row" }}>
+              {assets.map((asset, index) => (
+                <Fragment key={asset.symbol}>
+                  {index ? <Text color="whiteish500">/</Text> : null}
```
Comment (vkulinich-cl) [inline src/sections/pools/stablepool/details/PoolDetails.tsx:52]: what is it supposed to render?

File: src/sections/pools/stablepool/details/PoolDetails.tsx [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+            gap: 8,
+            align: ["end", "start"],
+            width: ["auto", 118],
+          }}
+        >
+          <Text fs={13} color="basic400">
+            {t("liquidity.asset.details.fee")}
+          </Text>
+          <Text lh={22} color="white" fs={18}>
+            {tradeFeeDisplay}%
```
Comment (vkulinich-cl) [inline src/sections/pools/stablepool/details/PoolDetails.tsx:71]: use `value.percentage` translation

File: src/components/Modal/Modal.styled.ts (hunk 4 above)
Comment (kolebjak) [reply src/components/Modal/Modal.styled.ts:97]: wasn't working well with currency reserves design, it looks according to design without this shadow tbh 


Screenshot 2023-08-21 at 11 49 35

File: src/sections/pools/stablepool/details/PoolDetails.tsx (hunk 6 above)
Comment (kolebjak) [reply src/sections/pools/stablepool/details/PoolDetails.tsx:52]: It is rendering `/` (slash) between assets 

Screenshot 2023-08-22 at 7 58 09

**Comment by vkulinich-cl** [comment]
Screenshot 2023-10-09 at 12 42 55

**Comment by vkulinich-cl** [comment]
Screenshot 2023-10-09 at 12 43 30
Is this a new design or styles are broken a little bit?

**Comment by vkulinich-cl** [comment]
Screenshot 2023-10-09 at 13 16 40
it doesn't look good on my laptop, but I think we can figure it out in next PR

**Comment by vkulinich-cl** [comment]
Screenshot 2023-10-09 at 13 20 58
Icon is missing and and no space between value and symbol within Initial value

**Comment by vkulinich-cl** [comment]
only those styling issues, otherwise looks good

**Review by vkulinich-cl — APPROVED** [review]

**Pull request merged**